modules/nf-core/cobrameta/tests/main.nf.test

```diff
@@ -0,0 +1,123 @@
+nextflow_process {
+
+    name "Test Process COBRAMETA"
+    script "../main.nf"
+    process "COBRAMETA"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "cobrameta"
+    tag "gunzip"
+
+    setup {
+        run("GUNZIP") {
+            script "../../gunzip/main.nf"
+            process {
+                """
+                input[0] = Channel.of(
+                    [
+                        [ id:'test' ],
+                        file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/fasta/test1.contigs.fa.gz', checkIfExists: true)
+                    ]
+                )
+                """
+            }
+        }
+    }
+
+    test("contigs.fasta.gz & coverage.tsv & viral_contigs & bam") {
+
+        when {
+            process {
+                """
+                input[0] = GUNZIP.out.gunzip
+                input[1] = Channel.of(
+                    [
+                        [ id:'test'],
+                        [
+                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/coverage/test1_contigs.coverage.stats.txt', checkIfExists: true)
+                        ]
+                    ]
+                )
+                input[2] = Channel.of(
+                    [
+                        [ id:'test'],
+                        [
+                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/tsv/test1_contigs_ids.tsv', checkIfExists: true)
+                        ]
+                    ]
+                )
+                input[3] = Channel.of(
+                    [
+                        [ id:'test'],
+                        [
+                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/bam/test1_contigs.sorted.bam', checkIfExists: true)
+                        ]
+                    ]
+                )
+                input[4] = 'metaspades'
+                input[5] = 27
+                input[6] = 127
+                """
+            }
+        }
+
+        then {
+            assertAll (
+                { assert process.success },
+                { assert snapshot(
```

Comment:
Can you check some of th eoptional files here aswell? Maybe the ones you expect to be produced by your test input. You could check fasta headers.